MANIFEST.in

```diff
@@ -5,7 +5,6 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 include LICENSE
-include Makefile
```

Comment:
This PR can be closing #304 provided we don't use `make` anymore anywhere? Please check and amend commit log...